c/httpserver.c

```diff
@@ -3451,23 +3453,26 @@ static int handleHttpService(HttpServer *server,
       request->authenticated = serviceAuthNativeWithSessionToken(service,request,response,&clearSessionToken, &authResponse);
       break;
     }
-    int size = sizeof server->authHandler / sizeof server->authHandler[0];
-    for (int i = 0; i < size; i++) {
-      if (server->authHandler[i] != NULL) {
-        if (strcmp(server->authHandler[i]->type, SERVICE_AUTH_NATIVE_WITH_SESSION_TOKEN) == 0)
-        {
-          int rc = service->server->authHandler[i]->authFunction(conversation, request, service, response);
-          if (rc != 0) {
-            respondWithError(response, HTTP_STATUS_UNAUTHORIZED, "Not Authorized");
+    if (request->authenticated) {
+      int size = sizeof(server->authHandler)/sizeof(server->authHandler[0]);
```

Comment:
Nitpick, but is there a reason not to have blanks around `/`? The previous version had them, so does the rest of the new code (around operators).